examples/mobile/UIComponents/components/etc/progress/index.html

```diff
@@ -22,6 +22,12 @@ <h1>Progress</h1>
 			<li class="ui-li-anchor">
 				<a href="progress-bar-indeterminate/progress-bar-indeterminate.html">Progress bar Indeterminate</a>
 			</li>
+			<li class="ui-li-anchor">
```

Comment:
Is this proper that "Progress" is placed in "etc" ?
Progress has own category number 8 in guideline.